src/recEventTracker.cpp

```diff
@@ -282,6 +310,14 @@ void RecEventTracker::saveOfflineEventLog(uint64 timestampFrom, uint64 timestamp
             root.appendChild(currObj);
         }
     }
+    for (size_t i = 0; i < messages.size(); i++)
+        if (messages[i].timestamp >= timestampFrom && messages[i].timestamp <= timestampTo)
```

Comment:
Should this range inclusive on both sides? I think it should be low inclusive high exclusive.

Reply:
I think it is better like this, surely prevents data loss. I added a comment on it for the future though